ru/passwords.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    "password" => "Пароль должен быть не менее шести символов и совпадать с подтверждением.",
+    "password" => "Пароль должен содержать не менее шести символов и совпадать с полем подтверждения пароля.",
```

Comment:
That's exactly what [bikeshedding](https://en.wiktionary.org/wiki/bikeshedding) is. Instead of focusing on missed translations time was spent on *making it pleasant to someone's ear*